src/main/java/core/basesyntax/service/impl/FileWriteServiceImpl.java

```diff
@@ -1,27 +1,29 @@
 package core.basesyntax.service.impl;
 
-import core.basesyntax.model.Fruit;
 import core.basesyntax.service.FileWriterService;
 import java.io.BufferedWriter;
 import java.io.FileWriter;
 import java.io.IOException;
-import java.util.List;
+import java.util.Map;
 
 public class FileWriteServiceImpl implements FileWriterService {
```

Comment:
This service should only be responsible for data writing operations. Don't do any additional logic here. Please take as an input the results of ReportService call. Also, you shouldn't have hardcoded the output file name. This output file name should be also passes as a parameter to the writeReport method